Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/HMGs/hmgs.yml

```
@@ -76,3 +165,35 @@
     steps: 2
     zeroVisible: false
   - type: Appearance
+  - type: BallisticAmmoProvider
+    mayTransfer: true
+    whitelist:
+      tags:
+      - CartridgeLightRifle # 762x39
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)